addons/maptools/stringtable.xml

```diff
@@ -429,6 +445,7 @@
             <Italian>Su</Italian>
             <Russian>Вверх</Russian>
             <French>Monter</French>
+            <German>Hoch</German>
```

Comment:
I think "nach oben" or just "oben" would be more appropriate here, but I only have partial context, so take it with a grain of salt.

Reply:
Makes more sense, changed it :D